src/components/Tooltip/Tooltip.module.scss

```diff
@@ -140,8 +139,8 @@
   }
 
   .closeButtonCircle {
-    width: var(--Space-12);
-    height: var(--Space-12);
+    width: var(--Space-32);
```

Comment:
This was incorrectly set in the first place but somehow Chrome ignored the width and height limit and showed the full X button anyway. Safari doesn't want to cope with this error.